modules/effects/arrow-effects-kotlinx-coroutines/src/test/kotlin/arrow/effects/DeferredTest.kt

```diff
@@ -127,5 +141,16 @@ class DeferredKTest : UnitSpec() {
         // Success
       }
     }
+
+    "awaitAll called on a Traverse instance of DeferredK<T> should return a Traverse instance of T" {
+      forAll(Gen.int(), {
+        runBlocking {
+          checkAwaitAll(ListK.applicative(), ListK.traverse(), it)
```

Comment:
If any of these fails, it just returns a boolean that's ignored. You have to && them!

If you were not using `forAll` you'd have to use `shouldBe` in checkAwaitAll, which throws an AssertionException instead.

Reply:
eek, school boy error. Thanks